hibernate-core/src/main/java/org/hibernate/dialect/SybaseDialect.java

```diff
@@ -73,13 +73,10 @@ public SybaseDialect() {
 
 	public SybaseDialect(DatabaseVersion version) {
 		super(version);
-		jtdsDriver = true;
 	}
 
 	public SybaseDialect(DialectResolutionInfo info) {
 		super(info);
-		jtdsDriver = info.getDriverName() != null
-				&& info.getDriverName().contains( "jTDS" );
```

Comment:
This change is problematic; if left in, a couple of hundred tests fail on Sybase due to `net.sourceforge.jtds.jdbc.ProtocolException`.
Any suggestions?

Reply:
Revert the change?

Reply:
hahah, yes i did that and all works again, but i meant any suggestion as to a type of fix that doesn't involve deaulting to jtds always or finding the cause (i could go and debug the driver, if you think it makes sense)

Reply:
This:
```
public SybaseDialect(DialectResolutionInfo info) {
		super(info);
		jtdsDriver = info.getDriverName() != null
				&& info.getDriverName().contains( "jTDS" );
}
```
causes CI failures on Sybase for this PR, not sure why. Remove it and no failures.
I'm ok with leaving the jtds compatible mode, just pointing out that CI complains.

Reply:
Solved that one now ... I think / hope